rasa/cli/arguments/x.py

```diff
@@ -1,6 +1,10 @@
 import argparse
 
-from rasa.cli.arguments.default_arguments import add_model_param, add_data_param
+from rasa.cli.arguments.default_arguments import (
+    add_model_param,
```

Comment:
we try to not import functions directly, but rather the module which contains them. E.g. 

```
from rasa.cli.arguments import default_arguments

default_arguments.add_model_param(...)
```

Could you please adapt your import and the old ones? :-)